Desktop Version

abdulrahmanshr75 opened the pull request:
- adjust the size of the pages
- adjust the size of the page elements
- added linters
- added new shapes for desktop view

abdulrahmanshr75 committed:
Update hompage

abdulrahmanshr75 committed:
Update My-Recent-Work-Section

changed the layout to desktop version

abdulrahmanshr75 committed:
fixed mobile verson

- adjust the size of the content to align

abdulrahmanshr75 committed:
Updated about me section

Developed the About me desktop version section

abdulrahmanshr75 committed:
Update contact me section

Adjust the whole section

abdulrahmanshr75 committed:
Fixed linters errors

abdulrahmanshr75 committed:
Merge branch 'main' into Desktop-version

abdulrahmanshr75 committed:
fixed linters

abdulrahmanshr75 committed:
Fixed linters

abdulrahmanshr75 committed:
fix linters

Cathella commented on index.html:
```diff
@@ -279,6 +282,42 @@ <h2 class="contact">Abdulrahmansh75@gmail.com</h2>
+      <div class="line"></div>
+      <div></div>
+      <footer class="footer">
+        <ul class="footer-social">
+          <li>
+            <a href="#" class="social-link"
+              ><img src="images/twitter.png" alt="twitter"
+            /></a>
+          </li>
+          <li>
+            <a href="#" class="social-link"
+              ><img src="images/Linkedinicon.png" alt="Linkedinicon"
+            /></a>
+          </li>
+          <li>
+            <a href="#" class="social-link"
+              ><img src="images/medium.png" alt="medium"
+            /></a>
+          </li>
```
Hello @abdulrahmanshr75  and @aeon9300, the job you have done so far on this project is clearly visible 💪🏻. Kudos to you 🍾. I however will require some small changes to make this portfolio perfect 💯 .

- [ ] I have noticed that the social icons at the bottom have almost no space between them and I think it will really be nice if you added some bit of padding to allow some breathing space. It will also help the users who would want to visit your social handles to click on them without getting worked out 🥺.

👉🏻 You can also compare designs below to see how much padding to add.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/4247740/145658189-bb29dd4f-5e9d-4868-8bd7-9455cca1b993.png) | ![image](https://user-images.githubusercontent.com/4247740/145658210-b1fffc5a-b95a-4a47-8959-e35943545517.png) |

Cathella reviewed (changes requested):
## 🔴 Status: Changes Requested 🛑

### Hi @abdulrahmanshr75  + @aeon9300 👋🏻, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Project Highlights 💪🏻

- No linter errors. ✔️
- Used correct Github flow. ✔️
- Good commit history ✔️
- Used media queries for the Desktop breakpoint in the CSS file ✔️
- Use both Flexbox and Grid in order to layout the elements on the page ✔️
- Mobile design still looks like expected ✔️
- The hamburger button is gone and Item Navigation is centered ✔️


## Required Changes ♻️

- [ ] [REQUIRED] I request that you take a second look at the contact form section and try to add styles that will match those defined in the Figma design. This is important because it adds adds a level of sophistication to the website making it interesting to visit.

- For instance try to give special attention to the active states of the form elements especially the border as seen below 👇🏻 

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/4247740/145657709-39e1f720-6866-4ee4-aa21-12b69d2da794.png) | ![image](https://user-images.githubusercontent.com/4247740/145657763-44c1d873-2710-424b-ae8e-a1ae820880bf.png) |


### _Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

abdulrahmanshr75 committed:
Changes

- took a second look at the contact form section and : 1- changed the on hover  and on focus border color in the text area.
- added a space between the social icons

elyor-doniyorov reviewed (approved):
Hi @abdulrahmanshr75  and @aeon9300, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
